docs/TROUBLESHOOTING.md

```diff
@@ -132,3 +133,61 @@ The **Service Account** must be declared as a job parameter.
 - **Terraform**: Use the input `service_account_email` from the [Dataflow Flex Job Module](../modules/dataflow-flex-job/README.md#inputs).
 
 For more details about Dataflow staging files see [Resource usage and management](https://cloud.google.com/dataflow/docs/guides/deploying-a-pipeline#resource-usage-and-management) documentation.
+
+### No matching distribution found for apache-beam==2.30.0
+
+An user deploys a new Dataflow job in the console and the job creation fails.
```

Comment:
consider "After deploying a new Dataflow job in the console, the job creation fails."